src/lib/paths.ts

```diff
@@ -172,6 +172,24 @@ export function getStringAfterLastSeparator(path: string): string {
   return path.split(window.electron.sep).pop() || ''
 }
 
+/**
+ * When you have an encodedURIComponent then you decode the URI component this can help split the path safely
+ * assumes /file/<encodedURIComponent>
+ * e.g '/file/%2Fhome%2Fkevin-nadro%2FDocuments%2Fzoo-modeling-app-projects%2Fbracket-1%2Fbracket.kcl'
```

Comment:
```suggestion
```

Nit, but this function actually does not want a router path like the one in this comment. It wants an already-decoded OS absolute file path with extension.

Reply:
lemme update this.